frontend/src/modules/entitieslist/components/Entity.css

```diff
@@ -33,10 +33,15 @@
 }
 .entity .translation-string {
     color: #AAAAAA;
-    font-style: italic;
     padding-left: 2%;
     text-align: start;
 }
+.entity .translation-string[data-script="Latin"],
+.entity .translation-string[data-script="Greek"],
+.entity .translation-string[data-script="Cyrillic"],
+.entity .translation-string[data-script="Vietnamese"] {
+    font-style: italic;
+}
```

Comment:
While you're at it, you should also add fix for [bug 1353135](https://bugzilla.mozilla.org/show_bug.cgi?id=1353135):

```css
#entitylist > .wrapper > ul > li p span.translation-string[lang="mk"],
#entitylist > .wrapper > ul > li p span.translation-string[lang="sr"] {
  font-family: "Ubuntu Regular";
}
```

Reply:
Done!